neuralmonkey/decoders/beam_search_decoder.py

```diff
@@ -231,9 +255,11 @@ def body(*args) -> BeamSearchLoopState:
             step = dec_loop_state.feedables.step - 1
 
             # mask the probabilities
-            # shape(logprobs) = beam x vocabulary
+            # shape(logprobs) = (batch*beam) x vocabulary
             logprobs = bs_state.prev_logprobs
 
+            input_beam_size = bs_state.input_beam_size
```

Comment:
spíš bych to šoupnul hnedka pod ten `step` na řádku 256. ale asi je to fuk.. možná by bylo nejlepší dát to ještě vejš, za `bs_output = `.. Ale možná je taky dobrý, aby to bylo tam, kde se to používá. Nebo by bylo dobrý to nepoužívat, protože je to použitý jen proto, že to má o 9 znaků míň, ale to je všechno.

Reply:
tech devet znaku navic mi rozbije formatovani :D

Reply:
To "Toho bych se asi nebál.. :-)" patří sem